tahoe_sites/zd_helpers.py

```diff
@@ -0,0 +1,16 @@
+"""
+Helpers used for Zero Downtime process. This file should be removed at the end
+"""
+from django.conf import settings
+
+
```

Comment:
Always provide a default, and preferably less mentions of the feature name to avoid misspelling.

def should_site_use_org_models():
    return settings.FEATURES.get('TAHOE_SITES_USE_ORGS_MODELS', False)

Reply:
@shadinaif `should_site_use_org_models` seems to be missing from the code.

Reply:
Thank you Omar. Yes, I'm adding it. Setting default is the standard, but I didn't want to add one so it fails if there is no explicit value. I've added a default now but I'm suggesting it to be `True` which is the actual default before activating functionalities of this repo. What do you suggest @OmarIthawi ?

Reply:
@shadinaif `True` works. As long as there's a good default it's good.